### PR TITLE
SpringBootAppDeploy_HelloWorld

### PR DESCRIPTION
**Pull request by ichha132**

### CODE REVIEW

**Comment by ichha132** [comment]
@googlebot I signed it!

File: backend/src/main/java/com/googleinterns.smb/consumer-backend/SMBApplication.java [hunk 1]
```diff
@@ -8,7 +8,11 @@
 import java.util.Map;
 @RestController
 @SpringBootApplication
-public class Practice2Application {
+<<<<<<< HEAD:backend/src/main/java/com/googleinterns.smb/consumer-backend/SMBApplication.java
+public class SMBApplication {
+=======
+public class SMBKiranaApplication {
+>>>>>>> a4d78c478aedc8db67c62642a7f8002bef85cc35:Practice2/src/main/java/com/example/Practice2/Practice2Application.java
```
Comment (pulkit-singhal) [inline backend/src/main/java/com/googleinterns.smb/consumer-backend/SMBApplication.java]: Fix the Git merge conflict

File: backend/src/main/java/com/googleinterns.smb/consumer-backend/SMBKiranaApplication.java [hunk 2]
```diff
@@ -8,15 +8,13 @@
 import java.util.Map;
 @RestController
 @SpringBootApplication
-<<<<<<< HEAD:backend/src/main/java/com/googleinterns.smb/consumer-backend/SMBApplication.java
-public class SMBApplication {
-=======
+<<<<<<< HEAD:backend/src/main/java/com/googleinterns.smb/consumer-backend/SMBKiranaApplication.java
```
Comment (pulkit-singhal) [inline backend/src/main/java/com/googleinterns.smb/consumer-backend/SMBKiranaApplication.java:11]: How >>>>>>> is still in the commit
Please remote the git diff lines

File: backend/src/main/java/com/googleinterns.smb/consumer-backend/SMBApplication.java (hunk 1 above)
Comment (ichha132) [reply backend/src/main/java/com/googleinterns.smb/consumer-backend/SMBApplication.java]: Done..?

File: backend/src/main/java/com/googleinterns.smb/consumer-backend/SMBKiranaApplication.java (hunk 2 above)
Comment (ichha132) [reply backend/src/main/java/com/googleinterns.smb/consumer-backend/SMBKiranaApplication.java]: Done

**Pull request merged**